shared/src/authorization/authorizationClientService.test.js

```diff
@@ -52,6 +52,15 @@ describe('Authorization client service', () => {
     ).toBeTruthy();
   });
 
+  it('should authorize a petitions serve petition', () => {
```

Comment:
"petitions clerk" instead of just "petitions"?

Reply:
Good catch!

Reply:
Updated!